pkg/node/node.go

```diff
@@ -353,7 +363,7 @@ func (no *node) monitorCFilters() {
 				continue
 			}
 
-			err = no.filtersDb.PutFilter(context.Background(), entry)
+			err = n.filtersDb.PutFilter(context.Background(), entry)
```

Comment:
no default: case?

How to close(ch) otherwhise?

Reply:
Purpose of for/select combo for channels(at least in this case, cant think of use case for default case ATM) is to listen to specific channels and block, in L354 there is quit chan to exit the loop.